### PR TITLE
To add and update the function name

### PR DESCRIPTION
**Pull request by xichengliudui**
To add and update the function name

### CODE REVIEW

File: pkg/framework/openapi/openapi.go [hunk 1]
```diff
@@ -78,7 +78,7 @@ func (d *document) LookupResource(gvk schema.GroupVersionKind) proto.Schema {
 	return d.models.LookupModel(modelName)
 }
 
-// Get and parse GroupVersionKind from the extension. Returns empty if it doesn't have one.
+// parseGroupVersionKind get and parse GroupVersionKind from the extension. Returns empty if it doesn't have one.
```
[the remark this answers is not in the document]
Comment (xichengliudui) [reply pkg/framework/openapi/openapi.go]: ParseGroupVersionKind is the function name，Do you have any questions?

[the remark this answers is not in the document]
Comment (apelisse) [reply pkg/framework/openapi/openapi.go]: Can you please fix the comment to read "parseGroupVersionKind get**s** and parse**s** ...", it's missing the "s" right now.

**Comment by xichengliudui** [comment]
@apelisse ，Thank you for your remind, I was too stupid

**Comment by apelisse** [comment]
/lgtm
/approve

Thank @xichengliudui !

**Pull request closed without merging**